src/components/VideoPage.js

```diff
@@ -3,81 +3,174 @@ import YouTube from "react-youtube";
+      try {
+        const response = await axios.get(
+          `https://videolib.amansethi00.repl.co/videos/${newVideoId}`
+        );
+        setVideo(response.data.video);
+      } catch (error) {
+        console.log({error});
+      }
     };
+    anonymousFunction();
   }, []);
+  const addToWatchedVideos = async () => {
+    try {
+      const response = await axios.post(
+        `https://videolib.amansethi00.repl.co/watchedVideos/${newVideoId}`,
+        "some",
+        {
+          headers: {
+            Authorization: `${localStorage?.getItem(
+              "username"
+            )}:${localStorage?.getItem("password")}`,
+          },
+        }
+      );
+      console.log({response});
+    } catch (error) {}
+  };
+  const addToLikedVideos = async (videoId) => {
+    try {
+      const response = await axios.post(
+        `https://videolib.amansethi00.repl.co/likedVideos/${videoId}`,
+        "data",
+        {
+          headers: {
+            Authorization: `${localStorage?.getItem(
+              "username"
+            )}:${localStorage.getItem("password")}`,
+          },
+        }
+      );
+      if (response.data.success === false) {
+        setError(response.data.message);
+      } else {
+        setSuccessMessage(response.data.message);
+      }
+      console.log(response);
+    } catch (error) {
+      setError(error.response.data.message);
+
+      console.log({error});
+    }
+  };
+  const playlistHandler = () => {
+    if (login) {
+      setShowPlayList(true);
+    } else {
+      setError("Please login to perform this action");
+    }
+  };
   return (
     <div className="videopage-container">
-      <div className="video-container">
-        <YouTube
-          videoId={newVideoId}
-          className="responsive-iframe"
-          opts={{
-            paddingTop: "0",
-            height: "390",
-            width: "700",
-            playerVars: {
-              autoplay: 1,
-            },
-          }}
-        />
-      </div>
-      <h2 className="card-head">{video.title}</h2>
-      <div className="flex row gray video-actions ">
-        <div>
-          <span className=" sm">{video.views}</span>
-          <span className=" sm">•{video.uploadDate}</span>
-        </div>
-        <div className="align-items-center row flex ">
-          <button
-            className="align-items-center row flex mg-right-half gray md"
-            style={{
-              color:
-                likes.filter((prev) => prev === video.id).length > 0 && "white",
-            }}
-            onClick={() => dispatch({type: "TOGGLE_LIKE", payload: video})}
-          >
-            <ThumbUpAltIcon />
-            <span>{video.totalLikes}</span>
-          </button>
-          <button
-            className="align-items-center row flex mg-right-half gray md"
-            onClick={() => setShowPlayList(true)}
-          >
-            <PlaylistAddIcon />
-            <span>SAVE</span>
+      {error && (
```

Comment:
This error toast displaces the whole page on error. Would be better if it was placed in same position as success one.